packages/tokens/src/constants/base.ts

```diff
@@ -101,4 +101,12 @@ export const baseTokens = {
     'LayerZero',
     'https://www.layerzero.foundation/',
   ),
+  usdt: new ERC20Token(
+    ChainId.BASE,
+    '0xfde4C96c8593536E31F229EA8f37b2ADa2699bb2',
+    6,
+    'USDT',
+    'Tether USDT',
+    'https://tether.to/',
+  ),
```

Comment:
Should add in `import { USDT } from './common'`

and make it usdt: USDT[ChainId.BASE],